doc/man/cgcreate.1

```diff
@@ -63,6 +63,11 @@ multiple times.
 .B -h, --help
 display this help and exit
 
+.TP
+.B -p <pid>
+uses the pid specified as the default process, in place of the idle
```

Comment:
Thinking out loud here: I would be tempted to not mention the idle process at all.  Perhaps something like this:

```
moves the specified \fB<pid>\fR into the
\fB<slice-name>.slice/<scope-name>.scope\fR.  Must be used in
conjunction with \fB-c\fR.
````

We should update the `-c` flag as well.  Perhaps something like:

```
.B -c, --scope
creates a new systemd scope. The cgroup name provided after the
\fB-g\fR flag must be of the form
\fB<slice-name>.slice/<scope-name>.scope\fR. If the slice
does not exist, systemd will create it. Libcgroup will place an
idle process in the scope's cgroup.procs file unless the \fB-p\fR
flag is provided
```

Reply:
Makes sense to update `-c` option, your rephrase looks better, I will make the man page change accordingly.